### PR TITLE
[hi] Add content/hi/docs/home/supported-doc-versions.md

### PR DESCRIPTION
**Pull request by verma-kunal**
Issue -> #29356

This PR is a part of ongoing` hi` localization effort

/language hi

### CODE REVIEW

File: content/hi/docs/home/supported-doc-versions.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: उपलब्ध दस्तावेज़ीकरण संस्करण
```
Comment (anubha-v-ardhan) [inline content/hi/docs/home/supported-doc-versions.md:2]:
```suggestion
title: उपलब्ध प्रलेखन संस्करण 
```

File: content/hi/docs/home/supported-doc-versions.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: उपलब्ध दस्तावेज़ीकरण संस्करण 
+content_type: custom
+layout: supported-versions
+card:
+  name: about
+  weight: 10
+  title: उपलब्ध दस्तावेज़ीकरण संस्करण
```
Comment (anubha-v-ardhan) [inline content/hi/docs/home/supported-doc-versions.md:8]:
```suggestion
  title: उपलब्ध प्रलेखन संस्करण 
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
We want to have some common words written the same way everywhere. We will soon make a Google sheet file with these words and their Hindi translations (just like the previous effort)

**Comment by verma-kunal** [comment]
@anubha-v-ardhan , i think `दस्तावेज़ीकरण` was a better word to be used here!! don't you think?

Screenshot 2021-08-13 at 8 22 52 PM

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM

**Comment by anubha-v-ardhan** [comment]
@verma-kunal Both `प्रलेखन` and `दस्तावेज़ीकरण` means the same, but `प्रलेखन` is a more formal alternative. Last localization attempt was also using `प्रलेखन` as they had mention in their Google sheets file.

**Comment by verma-kunal** [comment]
> @verma-kunal Both `प्रलेखन` and `दस्तावेज़ीकरण` means the same, but `प्रलेखन` is a more formal alternative. The last localization effort was also using `प्रलेखन` as they had mention in their Google sheets file.

sure, understood!!! Thank you 🙂

**Comment by mittalyashu** [comment]
@anubha-v-ardhan

> We want to have some common words written the same way everywhere. We will soon make a Google sheet file with these words and their Hindi translations (just like the previous effort)

I've created one a long time ago, it should also be pinned in hindi slack channel for Kubernetes 

https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0

**Comment by verma-kunal** [comment]
> @anubha-v-ardhan
> 
> > We want to have some common words written the same way everywhere. We will soon make a Google sheet file with these words and their Hindi translations (just like the previous effort)
> 
> I've created one a long time ago, it should also be pinned in hindi slack channel for Kubernetes
> 
> https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0

Yeah, we can add more into this as well !! Thank you for sharing

**Comment by anubha-v-ardhan** [comment]
Sorry @mittalyashu , we thought you are inactive and we didn't had edit access for that sheet. Glad to see you. We will use the existing file then : )

**Comment by mittalyashu** [comment]
@anubha-v-ardhan I tried continue working on it but gradually other Hindi contributors also left and then somewhere I also lost interest 😞

Glad to see more contributors coming back.

**Comment by anubha-v-ardhan** [comment]
@mittalyashu No worries, We are determined to finish the work this time. Please keep guiding us and reviewing our PRs and giving your valuable suggestions. Most of the folks in new team are newbies (including me)

**Comment by verma-kunal** [comment]
> @anubha-v-ardhan I tried continue working on it but gradually other Hindi contributors also left and then somewhere I also lost interest 😞
> 
> Glad to see more contributors coming back.

Glad to have you back with us @mittalyashu !! Would really look forward to your guidances & suggestions 😃✨

File: content/hi/docs/home/supported-doc-versions.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+title: उपलब्ध प्रलेखन संस्करण 
+content_type: custom
+layout: supported-versions
+card:
+  name: about
+  weight: 10
+  title: उपलब्ध प्रलेखन संस्करण 
+---
+
+इस वेबसाइट में कुबेरनेट्स के वर्तमान संस्करण और चार पिछले संस्करण के लिए दस्तावेज़ शामिल हैं।
```
Comment (mittalyashu) [inline content/hi/docs/home/supported-doc-versions.md:11]:
```suggestion
इस वेबसाइट में कुबेरनेट्स के वर्तमान और चार पिछले संस्करण के प्रलेखन शामिल हैं।
```

Comment (mittalyashu) [reply content/hi/docs/home/supported-doc-versions.md:11]: How about this?

Comment (mittalyashu) [reply content/hi/docs/home/supported-doc-versions.md:11]: @verma-kunal Can you make this change? And then we're good with this PR 🙂

**Comment by verma-kunal** [comment]
Done!! Thanks @mittalyashu for the review

**Review by anubha-v-ardhan — APPROVED** [review]

File: content/hi/docs/home/supported-doc-versions.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+title: उपलब्ध प्रलेखन संस्करण
```
Comment (mittalyashu) [inline content/hi/docs/home/supported-doc-versions.md:2]:
```suggestion
title: प्रलेखन के उपलब्ध संस्करण
```

This makes sentence more easy to read and understand.

File: content/hi/docs/home/supported-doc-versions.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---
+title: उपलब्ध प्रलेखन संस्करण 
+content_type: custom
+layout: supported-versions
+card:
+  name: about
+  weight: 10
+  title: उपलब्ध प्रलेखन संस्करण
```
Comment (mittalyashu) [inline content/hi/docs/home/supported-doc-versions.md:8]:
```suggestion
  title: प्रलेखन के उपलब्ध संस्करण
```

File: content/hi/docs/home/supported-doc-versions.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+title: उपलब्ध प्रलेखन संस्करण 
+content_type: custom
+layout: supported-versions
+card:
+  name: about
+  weight: 10
+  title: उपलब्ध प्रलेखन संस्करण 
+---
+
+इस वेबसाइट में कुबेरनेट्स के वर्तमान और चार पिछले संस्करण के प्रलेखन शामिल हैं।
```
Comment (mittalyashu) [inline content/hi/docs/home/supported-doc-versions.md:11]:
```suggestion
इस वेबसाइट में कुबेरनेट्स के वर्तमान और चार पिछले संस्करण के प्रलेखन हैं।
```

I think we can safely remove this word. What do you think?

**Review by mittalyashu — CHANGES_REQUESTED** [review]

File: content/hi/docs/home/supported-doc-versions.md (hunk 4 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/home/supported-doc-versions.md:2]: @verma-kunal
Can we commit these suggestions if you agree with the changes. We will be ready to merge this PR.

**Comment by anubha-v-ardhan** [comment]
@verma-kunal After commiting the suggestions, Can we squash the commits into one and rebase before merging please

**Comment by verma-kunal** [comment]
> @verma-kunal After commiting the suggestions, Can we squash the commits into one and rebase before merging please

sure!! would do that

File: content/hi/docs/home/supported-doc-versions.md (hunk 4 above)
Comment (verma-kunal) [reply content/hi/docs/home/supported-doc-versions.md]: I am not able to commit these!! its showing `Outdated suggestion`!! 
should i do this manually?

**Comment by anubha-v-ardhan** [comment]
/lgtm
/approve

Thank you @verma-kunal :)

**Pull request merged**